lib/duotone-filter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Duotone filter markup.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * SVG and stylesheet needed for rendering the duotone filter.
+ *
+ * @param  string $selector CSS selectors to apply the duotone to.
+ * @param  string $id       Unique slug for this duotone filter.
+ * @param  array  $values   R, G, and B values to filter with.
+ * @return string           Duotone stylesheet and SVG.
+ */
+function gutenberg_render_duotone_filter( $selector, $id, $values ) {
```

Comment:
Why is this thing not absorbed in the block support?

Reply:
It's needed later in the [image block render](https://github.com/WordPress/gutenberg/pull/26751/files/912b7d03348a0f2d747102177492be5c74eca72a#diff-1e4aa29ea183efda9c795271dc1acf6e8049355db104d0dca5b0ea5caf7f619cR29) and in any block that wants to implement duotone without block supports

Reply:
I was thinking that ideally, all blocks should rely on the block support and don't implement this adhoc. Is this possible?

Reply:
Yes, if we go with what you said in https://github.com/WordPress/gutenberg/pull/26751#discussion_r583430826, we can make that happen. 👍

Reply:
If this is getting moved exclusively to block supports, you can start reviewing the blocks supports parts unique to https://github.com/WordPress/gutenberg/pull/26752/files#diff-fd626ca219f3d805667d480eabb2d890a5a8d2e015b37f03cf8980edf1ae3d58 and https://github.com/WordPress/wordpress-develop/pull/984